client/state/data-layer/wpcom/jetpack-onboarding/index.js

```diff
@@ -116,13 +116,46 @@ export const saveJetpackOnboardingSettings = ( { dispatch, getState }, action )
 export const handleSaveSuccess = ( { dispatch }, { siteId, settings } ) =>
 	dispatch( saveJetpackOnboardingSettingsSuccess( siteId, settings ) );
 
-export const announceSaveFailure = ( { dispatch }, { siteId } ) =>
+export const announceSaveFailure = ( { dispatch }, { siteId } ) => {
 	dispatch(
 		errorNotice( translate( 'An unexpected error occurred. Please try again later.' ), {
 			id: `jpo-notice-error-${ siteId }`,
 			duration: 5000,
 		} )
 	);
+};
+
+export const retryOrAnnounceSaveFailure = ( { dispatch }, action ) => {
+	const MAX_WOOCOMMERCE_INSTALL_RETRIES = 2;
```

Comment:
We don't need to define this one on every function call; rather we can move it outside, and possibly export it to use it in tests.